webpack/components/AnsibleHostDetail/components/JobsTab/RecurringJobsTable.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { translate as __ } from 'foremanReact/common/I18n';
+
+import RelativeDateTime from 'foremanReact/components/common/dates/RelativeDateTime';
+
+import {
+  TableComposable,
+  Thead,
+  Tbody,
+  Tr,
+  Th,
+  Td,
+} from '@patternfly/react-table';
+
+import withLoading from '../../../withLoading';
+import { decodeId } from '../../../../globalIdHelper';
+
+const RecurringJobsTable = props => {
+  const columns = [__('Description'), __('Schedule'), __('Next Run')];
+
+  return (
+    <React.Fragment>
+      <h3>Scheduled recurring jobs</h3>
+      <TableComposable variant="compact">
+        <Thead>
+          <Tr>
+            {columns.map(col => (
+              <Th key={col}>{col}</Th>
+            ))}
+          </Tr>
+        </Thead>
+        <Tbody>
+          {props.jobs.map(job => (
+            <Tr key={job.id}>
+              <Td>
+                <Link to={`/job_invocations/${decodeId(job.id)}`}>
```

Comment:
This doesn't work, it redircts to `http://<foreman>/experimental/hosts/<host_fqdn>#/job_invocations/<job_id>`
Would this help?
```suggestion
                <Link to={`job_invocations/${decodeId(job.id)}`}>
```